config/audit.php

```diff
@@ -49,6 +76,8 @@
     |
     */
     'resolver' => [
+        'user_id'    => OwenIt\Auditing\Resolvers\UserIdResolver::class,
```

Comment:
Just one resolver, since both are doing pretty much the same thing, just calling two different methods (`->getMorphClass()` and `->getAuthIdentifier()`).